Description: various fixes of configure.ac to remove warnings
Author: Sergio Vavassori <vavassori@gmail.com>
Forwarded: no
Last-Update: 2024-04-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/scangearmp/configure.ac
+++ b/scangearmp/configure.ac
@@ -1,6 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT(configure.in)
+AC_INIT([scangearmp],[1.60-1])
+AC_CONFIG_SRCDIR(configure.ac)
 
 AC_ARG_ENABLE(libpath)
 if test "${enable_libpath}" = ""; then
@@ -12,15 +13,13 @@ AC_SUBST(CNMSLIBPATH)
 
 AC_SUBST(enable_libpath)
 
-AM_INIT_AUTOMAKE(scangearmp, 1.60-1)
-AM_CONFIG_HEADER(config.h)
-AC_PROG_LIBTOOL
+AM_INIT_AUTOMAKE
+LT_INIT
 AM_MAINTAINER_MODE
 
-AC_ISC_POSIX
+AC_SEARCH_LIBS([strerror],[cposix])
 AC_PROG_CC
-AM_PROG_CC_STDC
-AC_HEADER_STDC
+
 AC_PROG_RANLIB
 
 pkg_modules="gtk+-2.0 >= 2.6.0"
@@ -68,11 +67,11 @@ else
   AC_DEFINE_UNQUOTED(PACKAGE_LOCALE_DIR, "${prefix}/${DATADIRNAME}/locale")
 fi
 
-AC_OUTPUT(
-Makefile
+AC_CONFIG_FILES([Makefile
 po/Makefile.in
 backend/Makefile
 src/Makefile
 include/Makefile
-etc/Makefile)
+etc/Makefile])
+AC_OUTPUT
 
